src/unix/bsd/freebsdlike/freebsd/mod.rs

```diff
@@ -5406,6 +5411,15 @@ extern "C" {
         infotype: *mut ::c_uint,
         flags: *mut ::c_int,
     ) -> ::ssize_t;
+
+    pub fn timerfd_create(clockid: ::clockid_t, flags: ::c_int) -> ::c_int;
```

Comment:
Nit: the header and man page both define `clockid` as `int`, not `clockid_t`, though those are the same type.  Is the difference intentional?

Reply:
no I just misread.